File: Practica-CodigosPython/Ej15.py
#Hallar la división entera de 2 números, sin utilizar el operador “división”, 
#Sugerencia: Restas sucesivas
n1=int(input("Introducir el dividendo: "))
n2=int(input("Introducir el divisor:"))
c=0
while n1 >= n2:
    n1=n1-n2;c=c+1
print(c)

File: Practica-CodigosJava/Ej15.java
//Hallar la división entera de 2 números, sin utilizar el operador “división”, 
//Sugerencia: Restas sucesivas
import java.util.Scanner;
public class Ej15 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Introducir el dividendo: ");
        int n1 = scanner.nextInt();
        System.out.print("Introducir el divisor: ");
        int n2 = scanner.nextInt();
        int c = 0;
        while (n1 >= n2) {
            n1 = n1 - n2;
            c++;
        }
        System.out.println(c);

        scanner.close();
    }
}